From 1dedceae80ba46331d5bc03297e0820d9c3b05e4 Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Sun, 12 May 2024 06:30:30 -0500
Subject: [PATCH] Fix order in pyproject.toml

---
 pyproject.toml | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index 180f36f..3077f25 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -45,15 +45,15 @@ Homepage = "https://github.com/pyiron/pyiron_ontology"
 Documentation = "https://pyiron-ontology.readthedocs.io"
 Repository = "https://github.com/pyiron/pyiron_ontology"
 
+[tool.setuptools.packages.find]
+include = ["pyiron_ontology*"]
+
+[tool.setuptools.dynamic]
+version = {attr = "pyiron_ontology.__version__"}
+
 [tool.versioneer]
 VCS = "git"
 style = "pep440-pre"
 versionfile_source = "pyiron_ontology/_version.py"
 parentdir_prefix = "pyiron_ontology"
 tag_prefix = "pyiron_ontology-"
-
-[tool.setuptools.packages.find]
-include = [ "pyiron_ontology*",]
-
-[tool.setuptools.dynamic.version]
-attr = "pyiron_ontology.__version__"
